extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -373,11 +381,11 @@ export class BookTocManager implements IBookTocManager {
 			}
 		}
 		else if (element.contextValue === 'savedNotebook') {
+			// the notebook is part of a book so we need to modify its toc as well
+			const findSection = { file: element.book.page?.file?.replace(/\\/g, '/'), title: element.book.page?.title };
```

Comment:
It seems like this is the kind of thing that should probably go in a more common place such as updateTOC so that you're always sure you're writing out valid paths and every caller doesn't have to do this themselves.

Reply:
I'll be correcting this on my next PR.